mlrun/api/db/sqldb/models.py

```diff
@@ -33,6 +33,7 @@
 from sqlalchemy.orm import class_mapper, relationship
 
 from mlrun.api import schemas
+from mlrun.api.utils.sql_collation import SQLCollationUtil
```

Comment:
if the `SQLCollationUtil` will be here will it prevent the circular import ?

Reply:
Nope... I'll just have to move the `mlconf` import here and there'll still be the import loop